src/it/projects/profile-with-deps-inherit-parent-depMgmt-flatten-dep-all-oss-bom/verify.groovy

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
Is this right? It's not an Apache project though IANALL (I am not a license lawyer)

Reply:
@olamy please chime in -- we're adhering to the same license header as the other groovy files